newton/utils/render.py

```diff
@@ -17,6 +17,8 @@
 
 from collections import defaultdict
 
+from pxr import Usd, Gf, Sdf, UsdGeom
```

Comment:
Let's not add `usd-core` as a dependency to Newton but use deferred imports of `pxr` only inside the functions that need them, similar to `parse_usd`.